packages/url/src/filter-url-for-display.js

```diff
@@ -53,3 +53,14 @@ export function filterURLForDisplay( url, maxLength = null ) {
 		truncatedFile
 	);
 }
+
+/**
+ * Removes the protocol from a URL.
+ *
+ * @param {string} url - The URL to remove the protocol from.
+ *
+ * @return {string} The URL without the protocol.
+ */
+export function removeProtocol( url ) {
+	return url.replace( /^(?:https?:)\/\/(?:www\.)?/, '' );
```

Comment:
I'd expect this to use more or less the same regex as `isValidProtocol` or `getProtocol`.

Reply:
Hi @tyxla, I have made the changes as per your suggestion. Please take a look at it as per your convenience 🙇